backend/src/v2/driver/driver.go

```diff
@@ -429,6 +432,27 @@ func initPodSpecPatch(
 		userEnvVar = append(userEnvVar, k8score.EnvVar{Name: envVar.GetName(), Value: envVar.GetValue()})
 	}
 
+	if proxyConfig.GetHttpProxy() != "" {
+		glog.Infof("Setting http_proxy as %s", proxyConfig.GetHttpProxy())
+		userEnvVar = append(userEnvVar, k8score.EnvVar{Name: "http_proxy", Value: proxyConfig.GetHttpProxy()})
```

Comment:
Are these environment variable names meant to be lowercase?

Reply:
As discussed offline, added both lower and uppercase.